material/partials/languages/en.html

```diff
@@ -33,5 +33,6 @@
   "source.file.date.updated": "Last update",
   "source.file.date.created": "Created",
   "tabs.title": "Tabs",
-  "toc.title": "Table of contents"
+  "toc.title": "Table of contents",
+  "topbutton.title": "Back to Top"
```

Comment:
Thanks! Could you rename it into `top.title`? I know that's probably a little less self-explanatory, but it matches the style of `toc` and `tabs` with their feature flags:

- `tabs` -> `navigation.tabs`
- `toc` -> `navigation.toc`
- `top` -> `navigation.top`

I'm all for keeping consistency.

Reply:
Yup, done!